arrow/formatter.py

```diff
@@ -120,3 +120,75 @@ def _format_token(self, dt, token):
         if token == "W":
             year, week, day = dt.isocalendar()
             return "{}-W{:02d}-{}".format(year, week, day)
+
+    # COOKIE FORMAT: 'Thursday, 25-Dec-1975 14:15:16 EST'
+    def _format_cookie(self, dt):
+        # Date information
+        weekday = self._format_token(dt, "dddd")
+        day = self._format_token(dt, "DD")
+        month = self._format_token(dt, "MMM")
+        year = self._format_token(dt, "YYYY")
+
+        # Time information
+        hour = self._format_token(dt, "HH")
+        minute = self._format_token(dt, "mm")
+        second = self._format_token(dt, "ss")
+        timezone = self._format_token(dt, "ZZZ")
+
+        return "{}, {}-{}-{} {}:{}:{} {}".format(
+            weekday, day, month, year, hour, minute, second, timezone
```

Comment:
@rbandrews3 good work, would be nice to have this in a more succinct form.

Instead of doing every single one of the returns with a string format, arrow also has the format function, to which you can pass an entire formatted string and converts the tokens appropriately.

https://arrow.readthedocs.io/en/latest/#format

```arrow.utcnow().format('YYYY-MM-DD HH:mm:ss ZZ')```

Cheers!

Reply:
Yeah it might be good to do something similar to what we do in the `format` function itself: https://github.com/crsmithdev/arrow/blob/master/arrow/formatter.py#L28.

This way we can easily allow for translations to be inserted into the formatted time strings.